notebook/nbextensions.py

```diff
@@ -511,9 +511,9 @@ def validate_nbextension(require, logger=None):
     
     if logger:
         if warnings:
-            logger.warn(u"      - Validating: problems found:")
-            map(logger.warn, warnings)
-            map(logger.info, infos)
+            logger.warning(u"      - Validating: problems found:")
+            [logger.warning(warning) for warning in warnings]
+            [logger.info(info) for info in infos]
```

Comment:
I don't like using a listcomp when we don't care about the result. It's not actually any saving over doing:

```python
for warning in warnings: logger.warning(warning)
```